src/Plugin/Field/FieldWidget/PresetParagraphsWidget.php

```diff
@@ -210,7 +210,7 @@ public function formMultipleElements(FieldItemListInterface $items, array &$form
       '#required' => $this->fieldDefinition->isRequired(),
       '#field_name' => $field_name,
       '#cardinality' => $cardinality,
-      '#max_delta' => $max - 1,
+      '#max_delta' => is_int($max) ? ($max - 1) : '',
```

Comment:
LGTM @MdNadimHossain

Reply:
Technically don't need the brackets around `$max - 1` but they're not doing any harm :)

Reply:
yeah it is there for just visual purpose :p